src/components/ComponentPages/ChangePassword/index.tsx

```diff
@@ -27,13 +27,14 @@ const ChangePassword = () => {
     }
   };
   const onFinishFailed = (errorInfo) => {
-    console.log('Failed:', errorInfo);
+    console.log("Failed:", errorInfo);
   };
   return (
     <ChangePasswordUI
```

Comment:
I remember we discussed this yesterday to close in once if there is no text between the tags.